tests/tests.py

```diff
@@ -361,13 +381,13 @@ def tearDown(self):
 
     def test_load_state_successfully(self):
         """Tests if load state successfully"""
-        load_database = self.amity.load_state("amity_database")
-        self.assertEqual(load_database,
+        response = self.amity.load_state("amity")
```

Comment:
load_state() should load data from the database to the data structures. I think this test should involve confirming that some data from the database has indeed been added to the data structures.